src/schemas/projectConfig.schema.ts

```diff
@@ -151,6 +151,17 @@ const projectConfigJsonSchema = {
       },
       additionalProperties: false,
     },
+    cratesConfig: {
+      title: 'CratesTargetConfig',
+      description: 'Configuration options for the Crates target',
+      extends: { $ref: '#/definitions/targetConfig' },
+      properties: {
+        noDevDeps: {
+          type: 'boolean',
+        },
+      },
+      additionalProperties: false,
+    },
```

Comment:
Hm, does this actually work?
Based on the comment above, I really doubt it.
But we can probably revisit configuration validation some time soon.

Reply:
Not sure :) I only added it for completeness since I saw the others. Certainly doesn't hurt.